docs/digital-object-full-view.md

```diff
@@ -0,0 +1,32 @@
+# View Modes: Digital Object Full View
+
+`View modes` tell Archipelago Digital Objects (ADOs) how to be displayed to the world. This documentation will give a brief overview of the View mode `Digital Object Full View` and how it can be used when creating or editing ADOs.
```

Comment:
View Modes (or also know simple as 'Display'). I know it sound super stupid to have two names, but not sure why Drupal has these conflicting names in the menus. This is not a change request, just an open discussion for our call today